services/s3/src/it/java/software/amazon/awssdk/services/s3/crossregion/S3CrossRegionCrtIntegrationTest.java

```diff
@@ -20,14 +20,21 @@
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+import software.amazon.awssdk.http.apache.ApacheHttpClient;
+import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.ListObjectsRequest;
+import software.amazon.awssdk.services.s3.model.ListObjectsResponse;
 
 public class S3CrossRegionCrtIntegrationTest extends S3CrossRegionAsyncIntegrationTestBase {
     private static final String BUCKET = temporaryBucketName(S3CrossRegionCrtIntegrationTest.class);
 
     @BeforeAll
     static void setUpClass() {
-        s3 = s3ClientBuilder().build();
+        s3 = s3ClientBuilder().region(Region.AP_SOUTH_1).httpClient(ApacheHttpClient.create()).build();
```

Comment:
Question: why do we need to set ApacheHttpClient explicitly? If we have to, can we use `httpClientBuilder(ApacheHttpClient.builder())` so that the http client can be closed?

Reply:
Added by mistake just for testing , removed it now